ioc_finder/ioc_finder.py

```diff
@@ -15,6 +15,38 @@
 IndicatorDict = Dict[str, IndicatorList]
 # using `Mapping` b/c it is covariant (https://mypy.readthedocs.io/en/stable/generics.html#variance-of-generic-types)
 IndicatorData = Mapping[str, Union[IndicatorList, IndicatorDict]]
+POSSIBLE_DATA_TYPES = [
+    "urls", 
```

Comment:
Note: I'll be addressing the fact that there are different quotation marks used throughout this file in #222.